app/controllers/foreman_xen/snapshots_controller.rb

```diff
@@ -1,71 +1,74 @@
 module ForemanXen
   class SnapshotsController < ::ApplicationController
     helper :all
-    skip_before_filter  :verify_authenticity_token
+    skip_before_action :verify_authenticity_token
```

Comment:
You're right, and as _filter will only get removed in rails 5.1, I'm taking out this specific change.